js/react_native/android/src/androidTest/java/ai/onnxruntime/reactnative/TensorHelperTest.java

```diff
@@ -108,6 +108,35 @@ public void createInputTensor_int8() throws Exception {
     outputTensor.close();
   }
 
+  @Test
+  public void createInputTensor_uint8() throws Exception {
+    OnnxTensor outputTensor = OnnxTensor.createTensor(ortEnvironment, new byte[] {Byte.MIN_VALUE, 2, Byte.MAX_VALUE});
+
+    JavaOnlyMap inputTensorMap = new JavaOnlyMap();
+
+    JavaOnlyArray dims = new JavaOnlyArray();
+    dims.pushInt(3);
+    inputTensorMap.putArray("dims", dims);
+    inputTensorMap.putString("type", TensorHelper.JsTensorTypeUnsignedByte);
+
+    ByteBuffer dataByteBuffer = ByteBuffer.allocate(3);
+    dataByteBuffer.put(Byte.MIN_VALUE);
+    dataByteBuffer.put((byte)2);
+    dataByteBuffer.put(Byte.MAX_VALUE);
+    String dataEncoded = Base64.encodeToString(dataByteBuffer.array(), Base64.DEFAULT);
+    inputTensorMap.putString("data", dataEncoded);
+
+    OnnxTensor inputTensor = TensorHelper.createInputTensor(inputTensorMap, ortEnvironment);
+
+    Assert.assertEquals(inputTensor.getInfo().onnxType, TensorInfo.OnnxTensorType.ONNX_TENSOR_ELEMENT_DATA_TYPE_UINT8);
```

Comment:
`outputTensor` looks like it is set up in a similar way in `createInputTensor_int8`.
https://github.com/microsoft/onnxruntime/blob/03b57087cf704012468b4f097f27d264b868930f/js/react_native/android/src/androidTest/java/ai/onnxruntime/reactnative/TensorHelperTest.java#L83

but there, the type is compared to `TensorInfo.OnnxTensorType.ONNX_TENSOR_ELEMENT_DATA_TYPE_INT8`. how does that work?
https://github.com/microsoft/onnxruntime/blob/03b57087cf704012468b4f097f27d264b868930f/js/react_native/android/src/androidTest/java/ai/onnxruntime/reactnative/TensorHelperTest.java#L103

Reply:
Thanks for the comments, should have used positive numbers for test tensor data value.

Reply:
I see that createTensor() is given the type now, makes sense.